src/jit/gentree.cpp

```diff
@@ -7817,6 +7817,504 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     }
 }
 
+GenTreeOperandIterator::GenTreeOperandIterator()
+    : m_node(nullptr)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
+    , m_expandMultiRegArgs(false)
+    , m_state(-1)
+{
+}
+
```

Comment:
This should either use the existing `GetChild()` or replace it. We shouldn't be duplicating the handling of the special nodes all over the place.